addons/l10n_ar/data/account_tax_group.xml

```diff
@@ -59,7 +59,7 @@
 
     <!-- Others Taxes -->
 
-    <record model="account.tax.group" id="account.tax_group_taxes">
+    <record model="account.tax.group" id="tax_group_otros_impuestos">
         <field name="name">Otros Impuestos</field>
         <field name="sequence">20</field>
```

Comment:
@lordkrandel You are just renaming here.  And I don't see the point in removing the country on the tax group (which is not done here)

Reply:
Argentina is the only country that it's renaming and putting the country on `account.tax_group_taxes`, which is used as a default here:
https://github.com/odoo/odoo/blob/0ca0fdcee3d4610bcb14a0960dad8ad50f63884d/addons/account/models/account_tax.py#L68-L69

and that messes things up in the tests here when you have Argentina and another localization installed: 
https://github.com/odoo/odoo/blob/0ca0fdcee3d4610bcb14a0960dad8ad50f63884d/addons/account/models/account_tax.py#L130-L134

This otros_impuestos group is used only in demos in the code, I think, so I thought it was better to have a dedicated record for it and leave account.tax_group_taxes alone. 🤷🏻

Reply:
I agree with @lordkrandel